src/app/app.component.html

```diff
@@ -1,6 +1,21 @@
 
 <div class="wrapper">
-  <app-tree [tree]="items" class="first"></app-tree>
-  <app-tree [tree]="items" class="second"></app-tree>
+
+  <div class="header">
+    <form [formGroup]="myform">
+    <input type="text" class="header__search" formControlName="searchInput">
+    </form>
+  </div>
+
+  <div class="main">
+    <main class="main__content">
+      <form [formGroup]="myform">
```

Comment:
my form is pretty own naming, i suggest to change this one to treeForm or something else